src/main/java/cz/cvut/kbss/study/rest/PatientRecordController.java

```diff
@@ -62,15 +62,56 @@ public List<PatientRecordDto> getRecords(
 
     @PreAuthorize(
             "hasRole('" + SecurityConstants.ROLE_ADMIN + "') or @securityUtils.isMemberOfInstitution(#institutionKey)")
-    @GetMapping(value = "/export", produces = MediaType.APPLICATION_JSON_VALUE)
-    public List<PatientRecord> exportRecords(
+    @GetMapping(value = "/export", produces = {MediaType.APPLICATION_JSON_VALUE, Constants.MEDIA_TYPE_EXCEL})
+    public ResponseEntity<?> exportRecords(
             @RequestParam(name = "institution", required = false) String institutionKey,
-            @RequestParam MultiValueMap<String, String> params,
-            UriComponentsBuilder uriBuilder, HttpServletResponse response) {
+            @RequestParam(required = false) MultiValueMap<String, String> params,
+            UriComponentsBuilder uriBuilder, HttpServletRequest request, HttpServletResponse response) {
+        MediaType exportType = Stream.of(
```

Comment:
@LaChope here the way how to differentiate between JSON and EXCEL

Either use accept header or parameter in the query string:
```
public static final String EXPORT_TYPE_PARAM = "exportType";
public static final String MEDIA_TYPE_EXCEL = "application/vnd.ms-excel"
```

Reply:
Thus you need to use alternatively following:
`&exportType=application/vnd.ms-excel`
`&exportType=application/json`

Reply:
@blcham @kostobog Not sure that is works:

![image](https://github.com/kbss-cvut/record-manager/assets/57750426/b46bd4fc-3aac-4c26-b1e9-d574d21eff0c)

Reply:
ok, I'm not sure why it does not work, but you can try the `exportType` query parameter instead of the Accept header.

Reply:
@LaChope 
Record paging implementation does not support sorting without the `page` parameter.

Reply:
Removing the `sort` parameter fixed it. I can try with the `page` parameter but I do not know what odes it represent?

Reply:
The page parameter represents the index of the page. The page has `size` number of records. (request parameter, default is 25 records per page).

I am not sure what are you trying to do. There are three two things that I think makes sense to implement. 
1. Export all records matching the filter specified by the user. I think the user specifies the filter using the table. 
  - In this case the request should contain the same filter, e.g. min max date, institution, as the one used to filter the records in the table. `page`, `size` and `sort` parameters should be omitted. 
2. Export the records on the current page.  
  - In this case the request should contain the filter parameters as well as the and the `page`, `size`, and `sort` parameters.